### PR TITLE
osu!catch Ranking Criteria Update

### PR DESCRIPTION
**Pull request by Xinnoh**
https://osu.ppy.sh/community/forums/topics/770539

Added some guidelines copied over from standard mode
In addition, fixed an error where droplet should be drop

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 1]
```diff
@@ -47,15 +47,22 @@ Overall rules and guidelines apply to every kind of osu!catch difficulty.
...
 -   **Use the same slider tick rate on every difficulty** as it is a property of the music rather than the mapping. However, lower difficulties may use lower tick rates to reduce accuracy requirements for newer players, providing they still follow the rhythm of the song. Using high tick rates purely to increase score/combo/difficulty is senseless.
--	**Dashes and hyperdashes should not be used when the destination of the hyperjump is located near the left or right border of the play field.** This creates an uncomfortable movement as the catcher is forcibly stopped upon reaching the border of the playfield. Try to leave at least 16 osupixels of space between the end point of the hyperjump and the border of the play field, respectively at x:16 or x:496 at most.
+-	  **Dashes and hyperdashes should not be used when the destination of the hyperjump is located near the left or right border of the play field.** This creates an uncomfortable movement as the catcher is forcibly stopped upon reaching the border of the playfield. Try to leave at least 16 osupixels of space between the end point of the hyperjump and the border of the play field, respectively at x:16 or x:496 at most.**
```
Comment (Joehuu) [inline wiki/Ranking_Criteria/osu!catch/en.md]: Remove the `**` at the end.

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 2]
```diff
@@ -47,15 +47,22 @@ Overall rules and guidelines apply to every kind of osu!catch difficulty.
 -   **Your map must theoretically be possible to SS.** This means it must be possible to catch absolutely all fruits, including droplets.
 -   **Edge dashes must not be used in direct conjunction with hyperjumps.** This is because such patterns require especially precise movement and force an unreasonable restriction on accuracy required to catch them.
 -   **Each map must use at least two different combo colours which must not blend in with the map's background/storyboard/video.** This is so hit objects are always visible to the player.
+-   **Do not use keysounds without hitnormal support.** If sounds blend perfectly with the song, feedback for hitting notes is minimal.
+
```
Comment (Joehuu) [inline wiki/Ranking_Criteria/osu!catch/en.md]: Remove the extra line.

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 3]
```diff
@@ -47,15 +47,22 @@ Overall rules and guidelines apply to every kind of osu!catch difficulty.
...
+-   **Sliderends of extended sliders should be snapped according to the song's beat structure.** If the song is using a straight beat, 1/4, 1/8, and 1/16 should be used. If the song is using a swing beat, 1/6 or 1/12 should be used. If the song has a sound in a different snap from what was recommended, snapping to an actual beat always takes priority.
+-   **Slider tick rate should be set according to the song.** For example if your song only uses 1/3 snapping, using tick rate 2 or 4 would not be fitting.
+-   **Avoid using combo colours with ~50 luminosity or lower.** Dark colours impact readability of fruits with low background dim.
+-   **Avoid using combo colours with ~220 luminosity or higher.** They create bright pulses during kiai time which can be unpleasant to the eyes.
```
Comment (Joehuu) [inline wiki/Ranking_Criteria/osu!catch/en.md]: `kiai time` -> `Kiai Time`

File: wiki/Ranking_Criteria/osu!catch/en.md (hunk 1 above)
Comment (Xinnoh) [reply wiki/Ranking_Criteria/osu!catch/en.md]: fixed all

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 4]
```diff
@@ -47,15 +47,20 @@ Overall rules and guidelines apply to every kind of osu!catch difficulty.
...
 -   **Use the same slider tick rate on every difficulty** as it is a property of the music rather than the mapping. However, lower difficulties may use lower tick rates to reduce accuracy requirements for newer players, providing they still follow the rhythm of the song. Using high tick rates purely to increase score/combo/difficulty is senseless.
--	**Dashes and hyperdashes should not be used when the destination of the hyperjump is located near the left or right border of the play field.** This creates an uncomfortable movement as the catcher is forcibly stopped upon reaching the border of the playfield. Try to leave at least 16 osupixels of space between the end point of the hyperjump and the border of the play field, respectively at x:16 or x:496 at most.
+-	  **Dashes and hyperdashes should not be used when the destination of the hyperjump is located near the left or right border of the play field.** This creates an uncomfortable movement as the catcher is forcibly stopped upon reaching the border of the playfield. Try to leave at least 16 osupixels of space between the end point of the hyperjump and the border of the play field, respectively at x:16 or x:496 at most.
```
Comment (Joehuu) [inline wiki/Ranking_Criteria/osu!catch/en.md]: Revert the spacing at the beginning.

Comment (Xinnoh) [reply wiki/Ranking_Criteria/osu!catch/en.md]: disabled soft wrap -_-

Comment (Joehuu) [reply wiki/Ranking_Criteria/osu!catch/en.md]: It doesn't seem to be fixed. Right now, it parses this line as a code block.

**Pull request merged**